### ISSUE TITLE
How to deal with webdriver versioning issues?

### ISSUE THREAD

**Issue by smileservices** [clemfromspace/scrapy-selenium#112]
I'm getting `selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 88
Current browser version is 93.0.4577.63 with binary path /usr/bin/google-chrome` and I can't find a solution for it. I'm sure a lot encounter this. 

Is there a way to go over this and maybe get rid of specifying the driver executable? Maybe using `webdriver-manager` package?

**Comment by DResthal** [comment]
Honestly, I think the usage of the Selenium/standalone-chrome Docker container would be the best for long running code.

You can access it via the webdriver.Remote() function of Selenium, or set it up as your path for SELENIUM_COMMAND_EXECUTOR in this package, within your Scrapy settings.py.

This will help avoid chrome and driver versioning issues. 

Otherwise, you can use the ChromeDriverManager package, specifically the ChromeDriverManager.install() method

**Comment by nit-in** [comment]
> I'm getting `selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 88 Current browser version is 93.0.4577.63 with binary path /usr/bin/google-chrome` and I can't find a solution for it. I'm sure a lot encounter this.
> 
> Is there a way to go over this and maybe get rid of specifying the driver executable? Maybe using `webdriver-manager` package?

Did you find any solution to this problem?
I am also facing this issue.

**Comment by DResthal** [comment]
Sounds like you have the incorrect ChromeDriver version for your Chrome browser version.
I would double check both and that both are compatible, then double check that you're loading the driver you're expecting to load in your code.

**Comment by nit-in** [comment]
> Sounds like you have the incorrect ChromeDriver version for your Chrome browser version. I would double check both and that both are compatible, then double check that you're loading the driver you're expecting to load in your code.

Yeah..I have been updating chromedriver according to the chrome browser version, but I want to skip the version checking or have a solution that does not require manual updating of chromedriver.
